packages/create-next-app/create-app.ts

```diff
@@ -37,6 +37,7 @@ export async function createApp({
   skipInstall,
   empty,
   turbo,
+  app-api,
```

Comment:
kabab-case isn't a valid TS
Try `apiApi` or just `api`

Reply:
Thank you!